### PR TITLE
Add SASMA.ipynb and update README.md

### PR DESCRIPTION
**Pull request by kenty-k**
Summary
===
Added a new Jupyter Notebook file SASMA.ipynb.
Updated the README with information relevant to the research paper.
Branch name is add-sasma.

Review Request
===
Please check if the content of SASMA.ipynb is accurate.
Please check if the information in the README is appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ python inference.py \
 
 The results will be output to results/shadow_former+.
 
+## Training preprocessing
+For the training preprocesing , our proposed method# Assume shadow within the range of 40~100 uses SASMA.
+the code for SASMA is SASMA.ipynb.
```
Comment (Yuki-11) [inline README.md]: The first letter must be capitalized. Then please add the following text.

> For more information on SASMA, see the Citation section.

Comment (Yuki-11) [reply README.md]: The citation information will be added by me after I approve this pull request.

**Review by Yuki-11 — CHANGES_REQUESTED** [review]

**Comment by Yuki-11** [comment]
@kenty-k 
Thanks for your contributions! Please fix my request. 
Best,

**Comment by kenty-k** [comment]
@Yuki-11 
I have addressed your feedback and made the necessary changes.
- Edited lines 53 and 54 of README.md.
- Removed unnecessary comments and logs from SASMA.ipynb.
Please confirm that

**Comment by Yuki-11** [comment]
Thanks!

**Pull request merged**